service/dap/server_test.go

```diff
@@ -1771,6 +1773,8 @@ func TestSetFunctionBreakpoints(t *testing.T) {
 								}
 								continue
 							}
+							// Some function breakpoints may be in packages that have been imported and we do not control, so
+							// we do not always want to check breakpoints.
```

Comment:
do you mean breakpoint *lines*? hence the -1 used to ignore?

Reply:
Done.